src/utils.js

```diff
@@ -160,7 +160,8 @@ function objectFrozen(obj) {
 }
 
 function truncate(str, max) {
-  return !max || str.length <= max ? str : str.substr(0, max) + '\u2026';
+  if (typeof str !== 'string' || typeof max !== 'number' || max === 0) return str;
```

Comment:
`max` should never be dynamically resolved - it's taken off of global options. Maybe it could be better to just error here like you were instead of silently not truncating. 

Also - the places that truncate gets used, we probably don't want to return anything besides a string - right? If we were to pass through exception.value as an object, does the rest of the code even accept that? maybe it's better to return undefined or an empty string?
we could also coerce str to a string if it's not.

Reply:
Agree, I'll error out if `max` is not a number.

As for the second question, I think leaving it as is should be fine. Reason being:
a) we won't blow up when we send something else as instead of a string `value`
b) it'll be captured in the breadcrumb as `[object Object]` which may indicate that there's something funky going on and that could help developers find the culprit
c) in raw json event that we have available in Sentry, it should contain all the object attributes, which is basically everything someone might need to find out what was the error's value (in case it's some strange object like, in this case, jQuery Error - https://github.com/getsentry/raven-js/issues/1275)

Reply:
ah, that sounds good to me then! thanks for explaining